PublishParasoftResults/BuildApiClient.ts

```diff
@@ -71,6 +73,29 @@ export class BuildAPIClient {
         return [];
     }
 
+    async getCoberturaArtifactOfBuildById(buildId: number): Promise<BuildArtifact> {
+        return (await this.buildApi).getArtifact(this.projectName, buildId, COBERTURA_ARTIFACT_NAME);
+    }
+
+    async getCoberturaReportsOfArtifact(artifact: BuildArtifact): Promise<FileEntry[]> {
```

Comment:
Please refactor this method and `getSarifReportsOfArtifact` to reduce duplicated code.